tests/uefi_sb/utils.py

```diff
@@ -173,3 +173,20 @@ def check_vm_cert_md5sum(vm, key, reference_file):
     assert res.returncode == 0, f"Cert {key} must be present"
     reference_md5 = get_md5sum_from_auth(reference_file)
     assert hashlib.md5(res.stdout).hexdigest() == reference_md5
+
+def test_uefi_var_migrate(vm, source_host, dest_host):
```

Comment:
Needs to be named without a `test_` prefix to avoid interfering with pytest's collection process.